rcongui/src/components/useStyles.js

```diff
@@ -174,6 +174,12 @@ const useStyles = makeStyles(theme => ({
   popover: {
     pointerEvents: 'none',
   },
+  serverList:{
+    position: "absolute",
```

Comment:
Not sure if that's going to play well with the rest

Reply:
Works fine on my live version but your welcome to do it the correct way with grid :D

Reply:
Did you test in responsive mode on mobile? Cause that's usually when things go wrong with the header. It's currently a bit messed up already if the server name is too long. 

Let's merge this in "various" so that I can make some tweaks before it hits master.